File: java/naturalNumbers.java
import java.util.Scanner;
class Main 
{
  public static void main(String[] args)
  {
    System.out.print("How many natural numbers do you want to print?");
    Scanner scannerObject = new Scanner(System.in);
    int Count = scannerObject.nextInt();
    System.out.print("The natural numbers upto " + Count + " are");
    for(int Counter = 1; Counter <= Count; Counter++)
    {
      System.out.print(" " + Counter);
    }
    System.out.println(".");
  }
 
}

File: c#/naturalNumbers.cs
// A Hello World! program in C#.
using System;
namespace HelloWorld
{
    class Hello 
    {
        static void Main() 
        {
            int Count;
            Console.Write("How many natural numbers do you want to print?");
            Count = int.Parse(Console.ReadLine());
            Console.Write("The first " + Count + " natural numbers are");
            for(int Counter = 1; Counter <= Count; Counter++)
            {
              Console.Write(" " + Counter);
            }
            Console.WriteLinesum.pv(".");
        }
    }
}